index.js

```diff
@@ -72,6 +72,13 @@ function morgan (format, options) {
     deprecate('undefined format: specify a format')
   }
 
+  if (typeof fmt === 'string' && morgan[fmt] === undefined) {
+    if (fmt.split(':').length <= 1) {
+      deprecate('unidentified format: specify a proper format')
```

Comment:
So if this is just a deprecation, it shouldn't change the behavior. It should just print that the provide value is now deprecated, but should still work the same, otherwise it's semver major and will wait to wait to land.